app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -677,4 +677,70 @@
     <string name="detail_pinned_comment_view_description">釘選留言</string>
     <string name="background_player_already_playing_toast">已經在背景播放</string>
     <string name="leak_canary_not_available">LeakCanary 無法使用</string>
-</resources>
\ No newline at end of file
+
+    <string name="settings_category_sponsor_block_title">SponsorBlock （測試功能、第三方服務）</string>
+    <string name="settings_category_sponsor_block_categories_title">SponsorBlock 分類</string>
+    <string name="settings_category_sponsor_block_categories_summary">自訂要跳過的影片片段，以及它们在進度條上的顏色標記。</string>
+    <string name="settings_category_sponsor_block_categories_reset_colors_title">重設顏色</string>
+    <string name="settings_category_sponsor_block_category_enable_title">啟用</string>
+    <string name="settings_category_sponsor_block_category_color">進度條顏色</string>
+    <string name="settings_category_sponsor_block_category_sponsor_title">贊助商廣告</string>
+    <string name="settings_category_sponsor_block_category_sponsor_summary">付費宣傳內容、付費推薦內容和植入廣告</string>
+    <string name="settings_category_sponsor_block_category_intro_title">中間停頓/開頭動畫</string>
+    <string name="settings_category_sponsor_block_category_intro_summary">沒有實際內容的間隔。可以是暫停、靜態畫面、重複動畫</string>
+    <string name="settings_category_sponsor_block_category_outro_title">結束畫面/鳴謝</string>
+    <string name="settings_category_sponsor_block_category_outro_summary">鳴謝或當YouTube 結束畫面出現。沒有語音的結尾</string>
+    <string name="settings_category_sponsor_block_category_interaction_title">互動提醒 (訂閱)</string>
+    <string name="settings_category_sponsor_block_category_interaction_summary">當內容中途有一個簡短的提示來喜歡、訂閱或關注他們</string>
+    <string name="settings_category_sponsor_block_category_self_promo_title">非付費/自我推廣</string>
+    <string name="settings_category_sponsor_block_category_self_promo_summary">類似「贊助商廣告」，但是非付費或自我推廣。這包括有關商品、捐贈或與他人合作的資訊。</string>
+    <string name="settings_category_sponsor_block_category_non_music_title">音樂：非音樂部分</string>
+    <string name="settings_category_sponsor_block_category_non_music_summary">僅用於音樂影片。這包括音樂影片中的開頭或結尾</string>
+    <string name="no">否</string>
+    <string name="later">稍後</string>
+    <string name="invalid_color_toast">不正確的 HEX 顏色</string>
+    <string name="settings_category_feed_title">Feed</string>
+    <!-- SponsorBlock -->
+    <string name="sponsor_block">SponsorBlock</string>
+    <string name="sponsor_block_home_page_title">查看網站</string>
+    <string name="sponsor_block_home_page_summary">查看 SponsorBlock 的官方網站。</string>
+    <string name="sponsor_block_enable_title">跳過贊助</string>
+    <string name="sponsor_block_enable_summary">使用 SponsorBlock API 來自動跳過影片的廣告。目前只作用在 YouTube 的影片。</string>
+    <string name="sponsor_block_api_url_title">API 網址</string>
+    <string name="sponsor_block_api_url_summary">The url to use when querying the SponsorBlock API. This must be set for SponsorBlock to work.</string>
```

Comment:
Why isn't this translated?

I believe that if you leave out untranslated strings, it'll use the default language's string which is English. That'd be preferable.

Reply:
OK, finish.